client.go

```diff
@@ -338,6 +373,23 @@ func (c *Client) runOnce() error {
 		return err
 	}
 
+	if c.publishSettings.ConfirmMode {
+		// ConfirmMode is wanted, tell the amqp-server that we want to enable
+		// confirm-mode on this channel and start the confirms consumer.
+		err = outputCh.Confirm(false)
```

Comment:
Not a magic number but `false` here looks weird (setting confirm to `false`?!). Maybe this should be the `NoWait` field from the publisher settings (that's removed now) instead? If not, a variable. If not, a comment.

Reply:
I removed the `NoWait` field from everywhere because setting that to `True` makes is so that we don't wait for a channel to be created before we consume etc. So it just won't work reliably.

A lot of methods on an `amqp.Channel` takes a `noWait` argument. we could create a constant but I feel ok with just passing `false`.

Reply:
I think a constant is reasonable if you really want to get rid of the field. I guess it feels more open source thinking to me to have things configurable, especially while wrapping other tools that user might be used to deal with. However, if `amqp-rpc` behaves bad with this set to true I guess it's a good a idea to remove the field (and use a constant or comment).